src/content/audits/2020-gao-disaster-response.mdx

```diff
@@ -12,7 +12,7 @@ tags:
 
 import LinkButton from '../../components/LinkButton.astro';
 
-<LinkButton link="/files/audits/2020-gao-disaster-response.pdf" text="View Full Audit" />
+<LinkButton link="files/audits/2020-gao-disaster-response.pdf" text="View Full Audit" />
```

Comment:
FYI, I'm making  a mental note that we should fix this. This _should_ be an absolute url since the intent to link to a file starting at the root of the directory tree. Without the opening `/` it looks like it should be relative to the mdx file, which is confusing. Let's go ahead with it and come back to address the issue in the `LinkButton` component.